scarlet2/wavelets.py

```diff
@@ -0,0 +1,181 @@
+# from https://github.com/pmelchior/scarlet/blob/master/scarlet/wavelet.py
+
+import jax.numpy as jnp
+import numpy as np
```

Comment:
We should be able to do everything with jnp. That would remove any ambiguity on whether these are jax numpy variables or not.

Reply:
Sure, this is done